modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,260 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-running-dpdk-rootless-tap_{context}"]
+= Running the DPDK workload with kernel access
+
+It is desirable to allow certain types of traffic for example log messages to take a exception path from your DPDK application. To support this you need access to the kernel networking and the tap device plugin provides this capability. This allows the DPDK application to run as a non privileged user (rootless).
+
+.DPDK and TAP example configuration
+image::rootless_dpdk_example.png[DPDK and TAP plugin]
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the SR-IOV Network Operator.
+* Log in as a user with `cluster-admin` privileges.
+* Run `setsebools container_use_devices=true` as root on all nodes.
+
+.Procedure
+
+. Create the following `SriovNetworkNodePolicy` object, and then save the YAML in the `intel-dpdk-node-policy.yaml` file.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
```

Comment:
I think you need  needVhostNet: true flag installed. Better to sync with RnD about that.

Reply:
yes, it is needed as explained above.

Reply:
you can also ask a yaml used by @kononovn